tests/test_activate.py

```diff
@@ -73,6 +87,7 @@ def _write_entry_points(envs):
 mkdir -p {envs}/test2/bin
 """
 
+@unittest.skipIf(skip_tests, "conda-env not installed on path")
```

Comment:
Actually, I'd rather they failed. I don't see why conda-env wouldn't be installed as it's a dependency of conda, and if these tests are silently skipped, then we won't know if something in activate breaks.